Cargo.toml

```diff
@@ -43,6 +50,10 @@ wat = "1.0"
 name = "enarx-wasmres"
 path = "src/enarx-wasmres.rs"
 
+[[bin]]
+name = "enarx-wasmldr-tester"
+path = "src/enarx-wasmldr-tester.rs"
+
```

Comment:
This should be a test rather than a separate binary.

Reply:
Question: are tests supposed to be stand-alone (to be run by humans) or integrated (to be auto-run).  I've been doing only the former, so that we can do PoCs - if creating tests copes with both of these cases, I'm cool.

Reply:
We always want automated tests. Developers can run them with a simple `cargo test`. But they also get automatically integrated into our CI.